Refactor imports to be consistent with `edition = "2023_10"` (#473)
cairo-book/cairo-book: src/ch03-01-arrays.md

## Arrays

> An array is a collection of elements of the same type. You can create and use array methods without explicitly importing the `array::ArrayTrait` trait, as it is included in the prelude.

Reviewer: ```suggestion
An array is a collection of elements of the same type. You can create and use array methods by using the `ArrayTrait` trait from the core library.
```
